### PR TITLE
bpo-40798: Generate a different message for already removed elements

### PR DESCRIPTION
**Pull request by DahlitzFlorian**
@remilapeyre I did not include a news entry as it only affects the documentation.

https://bugs.python.org/issue40798

### CODE REVIEW

File: Doc/tools/extensions/pyspecific.py [hunk 1]
```diff
@@ -311,15 +312,26 @@ class DeprecatedRemoved(Directive):
...
+    _removed_label = 'Deprecated since version {deprecated}, removed in version {removed}'
 
     def run(self):
         node = addnodes.versionmodified()
         node.document = self.state.document
         node['type'] = 'deprecated-removed'
         version = (self.arguments[0], self.arguments[1])
         node['version'] = version
-        label = translators['sphinx'].gettext(self._label)
+        removed_major, removed_minor = map(int, self.arguments[1].split("."))
```
Comment (remilapeyre) [inline Doc/tools/extensions/pyspecific.py]: I think you can simply do 
```python

        env = self.state.document.settings.env
        current_version = tuple(int(e) for e in env.config.version.split('.'))
        removed_version = tuple(int(e) for e in self.arguments[1].split('.'))
        if current_version < removed_version:
            label = self._deprecated_label
        else:
            label = self._removed_label

        label = translators['sphinx'].gettext(label)
```

Then you won't need the `python_version`, `python_major` and `python_minor` class variables.

**Review by remilapeyre — CHANGES_REQUESTED** [review]
Thanks @DahlitzFlorian !

File: Doc/tools/extensions/pyspecific.py (hunk 1 above)
Comment (DahlitzFlorian) [reply Doc/tools/extensions/pyspecific.py]: Oh wow, that's awesome. Did not thought about solving it this way, thanks! I will update the PR.

File: Doc/tools/extensions/pyspecific.py [hunk 2]
```diff
@@ -35,6 +35,7 @@
 
 # Support for checking for suspicious markup
 
+import patchlevel
```
Comment (remilapeyre) [inline Doc/tools/extensions/pyspecific.py]:
```suggestion
```
This is not needed anymore

**Review by remilapeyre — CHANGES_REQUESTED** [review]

**Review by remilapeyre — APPROVED** [review]
Thanks @DahlitzFlorian, this looks great!

**Comment by DahlitzFlorian** [comment]
Thanks for the guidance @remilapeyre!

**Comment by JulienPalard** [comment]
Thanks @DahlitzFlorian I like it a lot!

Could you just add the new string to `Doc/tools/templates/dummy.html` so it get picked up by gettext and get translated?

**Comment by DahlitzFlorian** [comment]
Thanks for the feedback and pointing me at the translation part, @JulienPalard!

**Review by JulienPalard — APPROVED** [review]

**Comment by JulienPalard** [comment]
Thanks @DahlitzFlorian!

**Pull request merged**